simulator/messages/telemetry.py

```diff
@@ -1,39 +1,13 @@
-from enum import Enum
-
-
-class DroneActivity(int, Enum):
-    IDLE = 0
-    NAVIGATING = 1
-    REACHED_EDGE = 2
-    FOLLOWING_COMMAND = 3
+from simulator.position import Position
 
 
+# TODO: Josef, I heavily simplified this class, this will probably change the calls from the OMNeT++
+#  side, can you update them?
```

Comment:
I will check. However I think it should not make a difference right now since I import the Python Telemetry class and fill it. Right now I think its passed empty :)